src/main/conf/quattor-ccm

```diff
@@ -0,0 +1,240 @@
+# ${license-info}
+# ${developer-info}
+# ${author-info}
+# ${build-info}
+
+#
+# Set of tabcompletion function that can be used by other tools to access CCM data
+#
+
+# absolute path to tools, e.g. no fancy ls aliases and unnecessary PATH lookups
+_quattor_find_bin()
+{
+    local dir bin fn
+    fn="$1"
+    # use the "slow" sed here
```

Comment:
What's "slow" about it? Or is this a note it should be replaced by something faster? e.g. ${PATH//:/ }